### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by corpsee**
Documentation fixes

### CODE REVIEW

**Comment by lindelius** [comment]
@jmikola Useful PHPDoc changes that should be merged as soon as possible

**Comment by lindelius** [comment]
@corpsee Line 102 (after changes) in `src/Operation/CreateIndexes.php` should actually read `@param Server $server`

**Comment by corpsee** [comment]
@lindelius You are right, thx

File: src/Operation/CreateIndexes.php [hunk 1]
```diff
@@ -89,6 +100,8 @@ public function execute(Server $server)
      * command.
      *
      * @param Server $server
+     *
+     * @return mixed
```
Comment (jmikola) [inline src/Operation/CreateIndexes.php]: I realized while reviewing that this function's return value is never used. I've removed it entirely in https://github.com/mongodb/mongo-php-library/pull/188.

**Comment by jmikola** [comment]
Thanks for the contribution. I've selected parts of this PR and merged them to `v1.0` in https://github.com/mongodb/mongo-php-library/commit/bc64127f867f7f98c5201454b5d1a4fec63c50b6.

To explain why I've omitted other parts of the PR:

 * The doc blocks for private member variables aren't used for generating API documentation. Since these members are internal, I'd prefer to just name them sensibly and keep them self-documenting. In the cases of `private static` members, those are effectively internal constants whose value is immediately assigned (I don't see the value of prefixing them with `/** @var array */` or `/** @var int */`).
 * I am currently working through refactoring the GridFS component, so its code is in flux. I will be taking a final pass at the documentation for its public API once that refactoring is complete.

**Pull request closed without merging**